frames:
  meca:
    tcp: "tcp"
    world: "world"
    meca_mount: "meca_mount"
    home: "home"
    predispense: "predispense"
    service: "service"
    scene: "scene"
    substrate: "substrate"
    substrate_mount: "substrate_mount"
    cage: "cage"
    tool_mount: "tool_mount"